src/components/AddButton/types.ts

```diff
@@ -1,3 +1,5 @@
 export type AddButtonProps = {
-    action: () => void
+    action?: () => void,
+    type?: 'button' | 'link',
```

Comment:
it's better to make it NOT optional even though we added default value

Reply:
i`ve delete this option?
On this momment "AddButton" has used just in my "New User" page.
Possibly on other branch "AddButton" uses to